### PR TITLE
T06-Palindrome checker

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter pharse or word: ");
+         string input_word = Console.ReadLine ();
+         string word = input_word.Replace (" ", string.Empty);
```
Comment (MetaNaveen) [inline Program.cs:6]: Replace() and String Concatenation operations could be avoided using the Two Pointer technique.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+         string rev = "";
+         int w = word.Length - 1;
+         for (int i = 0; i <= w; w--) {
+            rev = rev + word[w];
```
Comment (MetaNaveen) [inline Program.cs:10]: Curly braces not required.

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+         }
+         if (word.ToLower () == rev.ToLower ()) {
+            Console.WriteLine ("palindrome");
+         } else Console.WriteLine ("not palindrome");
+      }
```
Comment (MetaNaveen) [inline Program.cs:15]: Could've used Ternary operator to print based on condition, in a single line.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (Priyanka15-12) [reply Program.cs]: Using two pointer technique.

**Review by MetaNaveen — APPROVED** [review]

**Comment by MetaNaveen** [comment]
Working as expected.

**Final notes:**

- Minor formatting issues.
- As per example in doc, apostrophe is not handled.
- Instead of "inputPhrase.Replace (" ", string.Empty)", you could only consider alphabets (ignoring all other characters) inside the loop.

**Pull request closed without merging**